README.md

```diff
@@ -50,13 +45,26 @@ Create a `.env` file in the root directory and put environment variables into th
 
 ```
 NODE_ENV=development
-API_BASE=http://localhost:4000/v1
-PORT=8080
 WEBPACK_DEV_SERVER_PORT=8081
-TITLE=ReCodEx
-ALLOW_NORMAL_REGISTRATION=true
-ALLOW_LDAP_REGISTRATION=false
-ALLOW_CAS_REGISTRATION=true
+LOGGER_MIDDLEWARE_VERBOSE=false
+LOGGER_MIDDLEWARE_EXCEPTIONS=true
+```
+
+## Configuration
+
+Compiled bundle properties can be modified by runtime configuration file. Sample
```

Comment:
Where the file should be located? Is there some example in repo?

Reply:
Final package location is `/opt/recodex-web/etc/env.json`.

Reply:
Well, that is after installation, but I also noticed the file in repo... describe this please in readme.